packages/openapi-to-graphql/test/example_gql_server.js

```diff
@@ -9,12 +9,12 @@ const express = require('express')
 const graphql = require('graphql')
 const { graphqlHTTP } = require('express-graphql')
 const app = express()
-const openAPIToGraphQL = require('../src/index')
+const openAPIToGraphQL = require('../dist/index')
```

Comment:
Why do we use `dist` here? In that case, we would need to build project to run the tests.

Reply:
@ardatan Can you clarify how to do that? Perhaps I am misunderstanding something.

Reply:
To clarify, this and `example_gql_server_ws.js` are written in JS, not TS.

Reply:
Another clarification, this isn't run by the test suite. This is strictly for manual testing.

Reply:
Jest sees `ts` files as modules so it resolves to `.ts` file even from `.js` files.

Reply:
Oh ok I thought it is called from jest :) We can use `ts-node` but it might be overhead. I think it is fine :)